modules/core/src/main/scala/org/scalasteward/core/scalafix/package.scala

```diff
@@ -51,6 +51,16 @@ package object scalafix {
         Nel.of("cats-core".r),
         Version("1.0.0"),
         Nel.of("github:fthomas/cats/Cats_v1_0_0?sha=update/scalafix")
+      ),
+      Migration(
+        "org.scalatest",
+        Nel.of("scalatest".r),
+        Version("3.1.0"),
+        Nel.of(
+          "https://raw.githubusercontent.com/scalatest/autofix/6168da0e2bd113872b7dcd22cad7688d97ef9381/3.0.x/rules/src/main/scala/org/scalatest/autofix/v3_0_x/RenameDeprecatedPackage.scala",
```

Comment:
If this is a rule for ScalaTest 3.0.x, it should be a separate migration with a 3.0.x `Version`.

Reply:
https://github.com/scalatest/scalatest/issues/1657 seems to suggest that this is a rewrite for ScalaTest 3.0.6.

Reply:
Yes.
But most of existing Steward-enabled repos should have been updated to  scalatest 3.0.8, if they merge Steward's  PR.
So I think migration for 3.0.6 are not performed for many repos, unless maintainers manually migrate.

Reply:
But what about new repos that are added to Scala Steward that use ScalaTest < 3.0.6? Those projects would benefit if this migration is applied in an update to the latest 3.0.x version. Maybe we should think about changing the `version` field of `Migration` to a range of versions. We should open a new ticket to discuss this.